lessons/12-css-pseudo-classes/index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+    <head>
+        <title>Pseudo Classes</title>
+    </head>
+    <style>
+        li:nth-child(3n){
+                color:green;
+                font-size: 20px;
+        }
+        li:first-child:hover{
+            background:red;
+            color:white;
+            font-size: 32px;
+        }
```

Comment:
Close! The font size is right here but we want to target the 2nd `<li>` on hover for the background and color